netbox_netprod_importer/vendors/cisco/nxos.py

```diff
@@ -105,15 +102,13 @@ def get_detailed_lldp_neighbours(self):
                 "local_port": local port name,
                 "hostname": neighbour hostname (if handled),
                 "port": neighbour port name,
-                "mgmt_id": neighbour id
+                "chassis_id": neighbour id
             }]
         """
-        cmd = "show lldp neighbors detail | json"
+        cmd = "show lldp neighbors detail" + self._driver_end_selection()
 
         cmd_output = self.device.cli([cmd])[cmd]
-        neighbours = json.loads(
-            cmd_output
-        )["TABLE_nbor_detail"]["ROW_nbor_detail"]
+        neighbours = self._correct_and_convert_to_dict(cmd_output)["TABLE_nbor_detail"]["ROW_nbor_detail"]
```

Comment:
Same than before about the 79char/l

Reply:
Of course I can, I just use the PEP8 standards.
If for you this is fundamentally changed.